classes/ezfindresultobject.php

```diff
@@ -10,7 +10,7 @@ class eZFindResultObject extends eZContentObject
     /*!
      \reimp
     */
-    function eZFindResultObject( $rows = array() )
+    function __construct( $rows = array() )
```

Comment:
You made the other such functions public but not this one. Why?

Reply:
Just overlooked that. Fixed now